tiup/tiup-component-dm-patch.md

```diff
@@ -72,8 +73,132 @@ You need to pack the binary package required for this command in advance accordi
 - Data type: `BOOLEAN`
 - This option is disabled by default with the `false` value. To enable this option, add this option to the command, and either pass the `true` value or do not pass any value.
 
-## Outputs
+## Example
 
-The execution log of tiup-dm.
+The following example shows how to apply `v5.4.0-hotfix` to the `v5.4.0` cluster deployed using TiUP. The operations might vary if you deploy the cluster using other methods.
```

Comment:
The following example shows how to apply `v5.4.0-hotfix` to the `v5.4.0` cluster deployed using TiUP,and other versions  are the same.